config/default/migrate_plus.migration.hs_courses.yml

```diff
@@ -16,6 +16,7 @@ migration_group: hs_courses
 label: 'Course importer'
 source:
   track_changes: true
+  delete_orphans: true
```

Comment:
Migrate or migrate plus doesn't already have this as an option somewhere?

Reply:
no. migrate was originally built to just consume one time data. if it was wrong it has a roll back option. but no syndication built in.

Reply:
https://www.drupal.org/node/2809433

Reply:
https://www.drupal.org/project/migrate_tools/issues/2863426

Reply:
ahh interesting. looks like the first link has a similar direction. but neither of these issues allows for a config option to sync vs just import. maybe at a later date this can be removed.

Reply:
the 2nd link is just a drush command. i dont wanna have to run a drush command for per migration per site.